src/frontend/src/utils/status.ts

```diff
@@ -91,16 +91,16 @@ const defaultFinalReviewModal: Modal = {
   cancelLabel: 'Cancel',
 };
 
-const checkReviewStatusDelegate = (pia: IPiaForm | null): boolean => {
+const checkReviewStatus = (pia: IPiaForm | null): boolean => {
+  // this function use to check if the review tab has any data, if so, show warning modal, otherwise
+  // display default modal
   if (
     pia &&
     (pia?.status === PiaStatuses.MPO_REVIEW ||
       pia?.status === PiaStatuses.FINAL_REVIEW) &&
-    pia?.hasAddedPiToDataElements === false &&
```

Comment:
I believe you need to have this check, If it is delegated you may not need CPO review
but when in full PIA we may need CPO review too. May be we can separate between delegated and non delegated in a switch/case statement?

Reply:
or we may be able to just add CPO and the logic would still work.